packages/core/src/index.js

```diff
@@ -1,4 +1,5 @@
-/* global process:readonly */
+/*eslint node/no-missing-import: "off" */
+/*node/prefer-global/process:readonly */
```

Comment:
Is this one correct syntax for ESLint?

https://github.com/mysticatea/eslint-plugin-node/blob/HEAD/docs/rules/prefer-global/process.md